resources/agent.proto

```diff
@@ -47,6 +47,7 @@ message TaskResponse {
   string token = 11;
   bool keep_alive_task = 12;
   string redact = 13;
+  string pre_signed_url = 14;
```

Comment:
maybe we should use `pre-signed_upload_url`.... just to make it more clear, what u think?

Reply:
Yeah, no problem